lib/CL/devices/remote/network_discovery.c

```diff
@@ -657,6 +658,26 @@ init_dht_discovery ()
 
 /*****************************************************************************/
 
+#if defined(ENABLE_REMOTE_DISCOVERY_ANDROID)
+
+/**
+ * Helper function used by android to add servers through the remote driver.
```

Comment:
Please run the formatter, the style seems wrong. Also separate the brief comment sentence with an empty line.